salt/metalk8s/salt/minion/installed.sls

```diff
@@ -4,6 +4,6 @@ include :
   - metalk8s.repo
 
 Install salt-minion:
-  {{ pkg_installed('salt-minion') }}
-    - require:
-      - test: Repositories configured
+{{ pkg_installed('salt-minion') }}
```

Comment:
This now looks incorrect, given unindent?

Reply:
Correction done.